e2e/helpers/tables.ts

```diff
@@ -239,33 +246,31 @@ const expectTableRowCorrect = async (
         .nth(colIndex)
         .textContent()
       await test.step(`confirm ${columnName}`, async () => {
-        const expectedText =
-          await test.step('format expected text', async () => {
```

Comment:
Removed the unnecessary `async`, which resulted in whitespace changes. No other changes in this chunk